app/services/service_provider_updater.rb

```diff
@@ -65,7 +65,7 @@ def parse_service_providers(body)
   end
 
   def dashboard_response
-    @_dashboard_response ||= HTTParty.get(url)
```

Comment:
RIP HTTParty 😢 (I'm not that sad about it)

Reply:
Consistency is key. No more parties! 😉